src/assets/_project/_blocks/layout/_typography.scss

```diff
@@ -1,6 +1,11 @@
-h2 {
-  line-height: 1.5em;
+h2, h3, h4 {
+  line-height: 1.4;
```

Comment:
Units have been dropped, did you mean 1.4em?

Reply:
Heading line-height values in SWE's underlying Bootstrap framework are unitless. Reading about line-height on [Mozilla's developer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) it says "The used value is the unitless number multiplied by the element's own font size. In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance." I made the call to keep SWE's values unitless as well.